docs/open-source-project/release-notes/tizen-5-0-m2.md

```diff
@@ -0,0 +1,468 @@
+  - Bug of rotation is fixed in Evas GL Engine.
+  - Added thread safety patch to Evas TBM Engine.
+  - Bug of image downscaling is fixed in Evas GL Engine.
+  - Added the patch is TBM Queue is used per TBM surface in Evas TBM Engine.
+-  SDL (Simple DirectMedia Layer)
+  - Bug of pause/resume in multi-windows is fixed.
+
+
+## UI framework
+
+### New and changed features
+
+- EFL
+  - Version 1.20 is upgraded to 1.21.
+- Harfbuzz
+  - Version 0.4.6 is upgraded to 1.8.1.
+- Freetype2
+  - Version 2.9 is upgraded to 2.9.1.
+- Text Input
+  - NLP (Natural Language Processing) API has been added.
+- Voice interaction
+  - Added new APIs to supporting multiple assistant.
+  - Added APIs which has platform privilege to supporting voice panel app. 
+  - Added APIs to supporting to send and to receive specific engine result.
+
+### Fixes
+
+- Fixed bugs.
+- CBHM (Clip Board History Manager) is now working properly.
+
+
+## Multimedia framework
+
+### New and changed features
+
+- Common
+  - Tizen Allocator & Bufferpool has been added.
+- Media Player
+  - Playback time related APIs in nanoseconds has been added.
+  - Setting zoom level with fov about spherical video API has been added.
+  - Video ROI (region of interest) API has been added.
+  - Support for resuming HTTP playback during buffering has been added.
+  - Support for cancellation of asynchronous preparing has been allowed.
+  - Support for some tags of HLS v20 has been added.
+  - The tizen allocator is applied to mediacodec.
+  - The tizen allocator & buffer pool have been applied to the following modules:
+    - Player, MediaCodec, Video360, VideoConverter
+    - TM1, TM2, TW2, ARTIK Codec Plugins
+    - Tizenwlsink
+- Camera
+  - Removed deprecated enums related with sound policy.
+  - Added new preview format for depth camera.
+  - Updated preview data structure for RGB format.
+- Media Controller 
+  - Added support for content type and age rating for the playing contents.
+  - Added support for icon setting for controller servers.
+  - Added support for capabilities of controller servers.
+  - Added support for search command.
+- Recorder
+  - Removed deprecated enums related with sound policy.
+- Sound Manager
+  - New APIs for USB audio output device are added.
+- Pulseaudio
+  - Added infrastructure of support for airplay(raop) device playback functionality.
+- OpenCV
+  - Version 2.4.9 has been upgraded to 3.4.1.
+  - Deep Neural Network (DNN) module has been enabled. 
+
+
+## Network and Connectivity
+
+### New and changed features
+
+- Network Monitoring
+  - Added intelligent-network-monitoring APIs.
+    - Added support for monitoring default connection capability.
+    - Added support for getting wireless link properties.
+    - Added support for watching network interface availability.
+    - Added support for providing various network status information for diagnosis.
+-  Wi-Fi Native CAPI
+  - Added a wifi-manager API for forgetting an AP asynchronously.

```

Comment:
Can we rephrase as:
Added a wifi-manager API for forgetting an access point (AP) asynchronously.

Reply:
I'll accept